Add typeArguments to ReferenceType.toString()

jiayihu opened the pull request:
First of all good job for the hard work on this project! ❤️ 
I'm using the `Converter` API to generate custom documentation for Angular components and I noticed that `typeArguments` are missing when calling `.toString()` on a `ReferenceType`.

aciccarello commented on src/lib/models/types/reference.ts:
```diff
@@ -17,7 +17,7 @@ export class ReferenceType extends Type
      * can be used to represent the type.
      */
-    name:string;
+    name:string; 
```
Looks like a space was accidentally added here.

jiayihu replied:
I actually thought that my editor removed a trailing space :D Will fix it!

aciccarello commented on src/lib/models/types/reference.ts:
```diff
@@ -115,10 +115,15 @@ export class ReferenceType extends Type
      * Return a string representation of this type.
      */
```
It might be helpful to add an example output string to the method docs

jiayihu replied:
You're right. I was also looking for some test for the class but I didn't it. Is there any? Thanks for your review (:

aciccarello replied:
You're welcome. I don't think there is currently a test set up. The travis build seems to have run fine.

jiayihu commented:
I updated after your suggestions. I added also a commit because many test files were updated by, I suppose, a `postinstall` script with the new Typedoc API domain.

aciccarello commented:
Great! Having the example output helps clarify what is going on.

I'm surprised that the domain wasn't changed before. Ideally that would have been done in a separate PR.

jiayihu commented:
No problem, I removed the last commit (:

jiayihu commented:
Any update? Are you @aciccarello allowed to merge PRs? You're helping a lot in issues too, would be sad to leave the project inactive.

jiayihu commented:
@blakeembrey thanks for merging (: do you have any thoughts about adding collaborators to the project to keep it alive if you don't have much time?

aciccarello commented:
@jiayihu We've had a few conversations on #256 and #378. I do think that we could do more in inviting people to be involved. Maybe we could add some information to the readme.

blakeembrey commented:
If someone is willing to take on the work, I'm certainly happy to add them as a maintainer and collaborator. I added @aciccarello for that reason 😄  He also linked to the related issues which should shed more light - I do hope to give this project to someone who can take it to the right place eventually, since that person is not currently me.
